src/bindings/c/tests/ov_preprocess_test.cpp

```diff
@@ -176,6 +176,161 @@ TEST(ov_preprocess, ov_preprocess_preprocess_steps_resize) {
     ov_core_free(core);
 }
 
+TEST(ov_preprocess, ov_preprocess_preprocess_steps_scale) {
+    ov_core_t* core = nullptr;
```

Comment:
We can apply INSTANTIATE_TEST_SUITE_P to put below similar tests together to avoid code duplication.